config/config.toml.example

```diff
@@ -198,8 +198,12 @@ record-db-qps = false
 max-procs = 0
 
 # Max memory size to use, 0 use the total usable memory in the machine.
+# The maximum memory limit for the Prepared Least Recently Used (LRU) caching
```

Comment:
Could you make this change in a separate PR because it's needed to cherry-pick to v2.1, v3.0, v3.1, and v4.0.

Reply:
sure